rspotify-model/src/idtypes.rs

```diff
@@ -26,7 +26,7 @@ pub trait PlayContextIdType: IdType {}
 macro_rules! sealed_types {
     ($($name:ident),+) => {
         $(
-            #[derive(Clone, Copy, Debug, PartialEq, Eq)]
+            #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
```

Comment:
Same here, Why would you like to derive the `Hash` trait?

Reply:
I needed it for something else and then forgot to remove it. But I think it makes sense anyway so one can have a `HashMap<&Id, ...>`, which makes sense to me.